### DISCUSSION TITLE
The 3D model cannot be visualized successfully with the function of 'gp.plot_3d'.

### DISCUSSION THREAD

**Discussion by chiyue520** [gempy-project/gempy#770] [Q&A]
Hello！
I have been using gempy for more than a year. Thank you very much for your team's efforts and great help for my scientific research work. However, when I was modeling with gempy today,  the following errors were reported :Number of scalars (0) must match either the number of points (60000) or the number of cells (55419).
![图片](https://user-images.githubusercontent.com/84581378/224015697-44454806-f228-438a-b6fa-085cc7f9dc14.png)
Could you support any help? Thank you.

**Comment by AlexanderJuestel** [comment]
Hello @chiyue520,

could you provide us with some more information? Which GemPy Version are you using? Which PyVista Version are you using? What have you done differently this time?

Cheers 
Alex

**Reply by chiyue520** [reply]
The following figure shows the version of the relevant package.
![图片](https://user-images.githubusercontent.com/84581378/224017166-17de86be-1e34-407f-b598-d7f1aff02448.png)
![图片](https://user-images.githubusercontent.com/84581378/224017280-58d644c4-611a-4615-8578-231e5f90da36.png)
And I have used the same set of data to model previously, and succeeded. But this time, I did not change the data itself,  the error occurred. I don't know where the problem occurred.

**Comment by AlexanderJuestel** [comment]
@chiyue520 do you still have the issue with the latest versions of GemPy and PyVista?
